src/Rendering/babylon.geometryBufferRenderer.ts

```diff
@@ -3,10 +3,22 @@ module BABYLON {
      * This renderer is helpfull to fill one of the render target with a geometry buffer.
      */
     export class GeometryBufferRenderer {
+        /**
+         * Represents the index of the position texture in the G-Buffer textures array
+         */
+        public positionIndex: number = 2;
+        /**
+         * Represents the index of the velocity texture in the G-Buffer textures array
+         */
+        public velocityIndex: number = 3;
+
         private _scene: Scene;
         private _multiRenderTarget: MultiRenderTarget;
         private _ratio: number;
         private _enablePosition: boolean = false;
+        private _enableVelocity: boolean = false;
+
+        private _previousWorldMatrices: { [index: string]: Matrix } = { };

```

Comment:
@deltakosh 
possibilities:
- Store a _previousWorldMatrix: Matrix on mesh class ?
- Keep this dictionary here ?